frontend/src/components/ItemCard.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "../styles/App.css";
+import {Card, Button} from "react-bootstrap";
+
+export interface ItemProp {
+    item: {
+        name: string,
+        price: string,
+        desc: string,
+        img: string
+    }
+}
+
+
+export default class ItemCard extends React.Component<ItemProp, {}> {
+    constructor(props : any){
+        super(props);        
+    }
+
+    render() {
+        return (
+            <Card style={{ width: '18rem' }}>
+                <Card.Img variant="top" src={require(`../../public/placeholder_assets/${this.props.item.img}`)} />
```

Comment:
@vong98 can these be made a static path instead of loading the file on each render.
you should be able to do something like `placholder_assets/<propname>.jpg`

Reply:
Would this make all the images the same on all the cards?

Reply:
Pushed a commit to fix it.
``` typescript
<Card.Img
        variant="top"
	src={`./placeholder_assets/${this.props.item.img}`}
/>
```
Just needed to build the path for each object